webmention.py

```diff
@@ -67,6 +68,14 @@ def discover(url, **requests_kwargs):
       logger.debug(f'Webmention discovery: got endpoint in tag: {endpoint}')
       return Endpoint(endpoint, resp)
 
+  # If we are not currently following a client-side redirect https://www.w3.org/TR/WCAG20-TECHS/H76.html
```

Comment:
Oh! I was thinking this kwarg would be an opt in for whether to follow client side redirects at all. Mind changing it to mean that?

Reply:
And thanks for the doc link!

Reply:
Changed